File: pingyougou_sellergoods_interface/src/main/java/com/pingyougou/sellergoods/service/BrandService.java
package com.pingyougou.sellergoods.service;

import com.pingyougou.pojo.PageResult;
import com.pingyougou.pojo.TbBrand;

import java.util.List;

/**
 * @author yin
 * @Date 2018/8/11 22:11
 * @Method
 */

public interface BrandService {
    List<TbBrand> findAll();

    public PageResult<TbBrand> findPage(int pageNum, int pageSize);

    void add(TbBrand tbBrand);

    void update(TbBrand tbBrand);
    TbBrand findOne(Long id);

    void delete(Long[] ids);

    PageResult<TbBrand> search(TbBrand tbBrand, int page, int rows);
}

File: pingyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service('brandservice',function ($http) {
    this.search=function(page,rows,searchEntity){
       return  $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
    }

    this.update=function (entity) {
        return $http.post('../brand/update.do',entity);
    }

    this.add=function (entity) {
        return $http.post('../brand/add.do',entity);
    }

    this.findOne=function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    }
    this.dele=function (ids){
        return $http.get('../brand/delete.do?ids='+ids);
    }
})